pkg/reconciler/pipelinerun/resources/pipelineref_test.go

```diff
@@ -205,7 +206,7 @@ func TestGetPipelineFunc(t *testing.T) {
 					PipelineRef:        tc.ref,
 					ServiceAccountName: "default",
 				},
-			})
+			}, []*v1alpha1.VerificationPolicy{})
```

Comment:
Could you add tests with len(verificationPolicies) > 0.

Reply:
it should be tested in the latest commit